src/main/java/com/zaxxer/hikari/pool/PoolBase.java

```diff
@@ -134,13 +134,9 @@ void quietlyCloseConnection(final Connection connection, final String closureRea
             logger.debug("{} - Closing connection {}: {}", poolName, connection, closureReason);
 
             try {
-               setNetworkTimeout(connection, SECONDS.toMillis(15));
-            }
-            catch (SQLException e) {
-               // ignore
-            }
-            finally {
-               connection.close(); // continue with the close even if setNetworkTimeout() throws
+               trySetNetworkTimeout(connection, SECONDS.toMillis(15));
+            } finally {
+               connection.close();
```

Comment:
I believe it is intentional to ignore any SQLException here. If the connection is already closed and setNetworkTimeout throws the exception because of this, then the exception is ignored. I don't think this code need to be changed.

Reply:
about the SQL exception that is ignored here, without concerning myself about the context of the discussion, I was thinking is the SQL exception not a log worthy event. Should it not be logged at least at trace level, or even a higher level be considered? 
Or perhaps I am mistaken?